assets/loadtest/etcd/certs/Makefile

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# output:
+#   ca-key.pem      : private key of the trusted CA
+#   ca-cert.pem     : self-signed CA cert
+#   client-crt.pem  : client cert signed by CA
+# 	client-key.pem  : private key of the client
+.PHONY:all
+all: ca-cert.pem server-cert.pem client-cert.pem
+	@rm -rf *csr
```

Comment:
Nit: I'm curious why removing CSR's is not a part of those respective targets-dependencies (seems like they're already removing intermediate files).

Reply:
This came from the original loadtest [branch](https://github.com/gravitational/ops/blame/fspmarshall/monster/loadtest/etcd/certs/Makefile#L10). Moved to the respective targets.